continuous_integration/home/coder/.local/bin/conda-utils

```diff
@@ -58,10 +58,12 @@ get_yaml_and_make_conda_env() {
     make_conda_env_from_yaml;
 }
 
-make_conda_env_using_legate_core() {
-    mamba create -n "${DEFAULT_CONDA_ENV:-legate}"
+install_legate_core_with_war() {
```

Comment:
Can you file a GH issue describing the issue and then reference the issue # here?

Reply:
Done. See #812.